hedera-mirror-rest/api/v1/openapi.yml

```
@@ -235,6 +235,58 @@ paths:
           $ref: "#/components/responses/NotFoundError"
       tags:
         - accounts
+  /api/v1/accounts/{idOrAliasOrEvmAddress}/allowances/nfts:
+    get:
+      summary: This API is currently not supported.Get non fungible token allowances for an account
+      description: |
+        ** This API is currently under development and is not supported yet **
+        Returns information for non fungible token allowances for an account.
+
+        ## Ordering
+        The order is governed by a combination of the account id and the token id values, with account id being the parent column.
+        The token id value governs its order within the given account id.
+
+        Note: The default order for this API is currently ASC. Account id can be the owner or the spender id depending on the owner flag.
+
+        ## Filtering
+        When filtering there are some restrictions enforced to ensure correctness and scalability.
+
+        **The table below defines the restrictions and support for the endpoint**
+
+        | Query Param   | Comparison Operator | Support | Description           | Example |
+        | ------------- | ------------------- | ------- | --------------------- | ------- |
+        | account.id    | eq                  | Y       | Single occurrence only. | ?account.id=X |
+        |               | ne                  | N       | | |
+        |               | lt(e)               | Y       | Single occurrence only. | ?account.id=lte:X |
+        |               | gt(e)               | Y       | Single occurrence only. | ?account.id=gte:X |
+        | token.id      | eq                  | Y       | Single occurrence only. Requires the presence of an **account.id** query | ?account.id=X&token.id=eq:Y |
+        |               | ne                  | N       | | |
+        |               | lt(e)               | Y       | Single occurrence only. Requires the presence of an **lte** or **eq** **spender.id** query | ?account.id=lte:X&token.id=lt:Y |
+        |               | gt(e)               | Y       | Single occurrence only. Requires the presence of an **gte** or **eq** **spender.id** query | ?account.id=gte:X&token.id=gt:Y |
```

Might be best to just copy the latest from the HIP for this API description.

```suggestion
        |               | lt(e)               | Y       | Single occurrence only. Requires the presence of an **lte** or **eq** **account.id** query | ?account.id=lte:X&token.id=lt:Y |
        |               | gt(e)               | Y       | Single occurrence only. Requires the presence of an **gte** or **eq** **account.id** query | ?account.id=gte:X&token.id=gt:Y |
```